### PR TITLE
Wip s3 tests python 3

### PR DESCRIPTION
**Pull request by alimaredia**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by alimaredia** [comment]
Will remove the last commit once my s3-tests python 3 port is passing on centos.

File: qa/suites/rados/basic/tasks/rgw_snaps.yaml [hunk 1]
```diff
@@ -26,16 +26,7 @@ tasks:
     - default.rgw.control
     - default.rgw.meta
     - default.rgw.log
-- s3readwrite:
+- s3tests:
```
Comment (tchaikov) [inline qa/suites/rados/basic/tasks/rgw_snaps.yaml]: does s3-tests offer (almost) the same coverage as s3tests-test-readwrite ?

Comment (alimaredia) [reply qa/suites/rados/basic/tasks/rgw_snaps.yaml]: For the sake of what's needed in the rados suite, yes

**Comment by alimaredia** [comment]
@tchaikov here is a run on the centos machine from a few days back: http://pulpito.ceph.com/amaredia-2020-01-13_19:43:41-rgw:verify-master-distro-basic-smithi/

I also removed the last commit that hard coded one of my branches, so this branch should be good to merge

**Comment by tchaikov** [comment]
`Reviewed-by: Kefu Chai <kchai@redhat.com>`

**Pull request merged**